newIDE/app/src/EventsSheet/EventsTree/Renderers/JsCodeEvent.js

```diff
@@ -186,9 +187,11 @@ export default class JsCodeEvent extends React.Component<
         tabIndex={0}
         style={textStyle}
       >
-        {parameterObjects
-          ? `, objects /*${parameterObjects}*/`
-          : ' /* Click here to choose objects to pass to JavaScript */'}
+        {parameterObjects ? (
+          <Trans>{`, objects /*${parameterObjects}*/`}</Trans>
```

Comment:
Same everywhere, remove all the brackets with the backticks :)

Reply:
Actually you're right, template literals are supported! 
It's in the Trans component test suite: https://github.com/lingui/js-lingui/blob/3afaceccb669b59ee2f5b42ee2e138646ccdb79d/packages/macro/test/jsx-trans.ts#L96-L107